core/src/identity/rsa.rs

```diff
@@ -107,6 +107,23 @@ impl PublicKey {
     }
 }
 
+impl fmt::Debug for PublicKey {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        fn to_hex(bytes: &[u8]) -> String {
```

Comment:
Well, please remove the function altogether :sweat_smile: 

```suggestion
```

Reply:
Yeah, kinda makes sense :blush: ...